pkg/hubble/metrics/api/context.go

```diff
@@ -281,10 +404,26 @@ func destinationIPContext(flow *pb.Flow) (context string) {
 	return
 }
 
+func getK8sAppFromLabels(labels []string) string {
+	ls := ciliumLabels.ParseLabelArrayFromArray(labels)
+	// iterate over the set of pod app labels we source from and return the first
```

Comment:
```suggestion
	// Iterate over the set of pod app labels we source from and return the first
```

Reply:
I'll fix this, but technically I only moved this function ;)

Reply:
Fixed